modules/gateway.tf

```diff
@@ -122,7 +116,9 @@ resource "aws_api_gateway_deployment" "gateway_deployment" {
     // API Changes won't get deployed, without a trigger in TF
     // See https://medium.com/coryodaniel/til-forcing-terraform-to-deploy-a-aws-api-gateway-deployment-ed36a9f60c1a
     // and https://github.com/terraform-providers/terraform-provider-aws/issues/162#issuecomment-475323730
-    change_trigger = sha256(data.template_file.api_swagger.rendered)
+  triggers = {
+    redeployment = sha1(jsonencode(aws_api_gateway_rest_api.gateway_api.body))
+  }
```

Comment:
@martwana it seems like the syntax here isn't working:

```console
❯ terraform plan
╷
│ Error: Incorrect attribute value type
│
│   on .terraform/modules/dce/modules/gateway.tf line 115, in resource "aws_api_gateway_deployment" "gateway_deployment":
│  115:   variables = {
│  116:     // API Changes won't get deployed, without a trigger in TF
│  117:     // See https://medium.com/coryodaniel/til-forcing-terraform-to-deploy-a-aws-api-gateway-deployment-ed36a9f60c1a
│  118:     // and https://github.com/terraform-providers/terraform-provider-aws/issues/162#issuecomment-475323730
│  119:   triggers = {
│  120:     redeployment = sha1(jsonencode(aws_api_gateway_rest_api.gateway_api.body))
│  121:   }
│  122:   }
│
│ Inappropriate value for attribute "variables": element "triggers": string required.
```

fixed (or removed the error on plan) by removing the "reployment" key:

```hcl
resource "aws_api_gateway_deployment" "gateway_deployment" {
  rest_api_id = aws_api_gateway_rest_api.gateway_api.id

  variables = {
    triggers = sha1(jsonencode(aws_api_gateway_rest_api.gateway_api.body))
  }

  lifecycle {
    create_before_destroy = true
  }
}
```